src/Dfe.Spi.GraphQlApi.Functions/host.json

```diff
@@ -0,0 +1,8 @@
+{
+    "version": "2.0",
+    "extensions": {
+        "http": {
+            "routePrefix": ""
+        }
+    }
+}
```

Comment:
Might want to include...

```
{
  "version": "2.0",
  "logging": {
    "applicationInsights": {
      "samplingSettings": {
        "isEnabled": false
      }
    },
    "logLevel": {
      "default": "Debug"
    }
  }
}
```

As this will mean we get all the logs.. I'm doing this on all function apps from now on...